src/test/librados/io.cc

```diff
@@ -440,3 +440,9 @@ TEST_F(LibRadosIoEC, XattrIter) {
   }
   rados_getxattrs_end(iter);
 }
+
+TEST_F(LibRadosIo, ZeroLenZero) {
```

Comment:
Nit: group this test together with other `LibRadosIo` tests, e.g. between `AppendRoundTrip` and `TruncTest`.

Reply:
@idryomov : Would it be better to move this into c_write_operations.cc? It's not clear to me how these tests are divvied up, but the other tests that use `rados_create_write_op()` are all in that file.

Reply:
Not sure.  c_write_operations.cc seems to be more focused on transaction semantics and multi-op requests, so keeping it here is probably fine.